<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="VWBXINGLIOA1" name="com.portal.base.testCase.All_Test_Case" tests="32" failures="1" timestamp="26 Nov 2019 06:21:27 GMT" time="11.165" errors="1">
  <testcase name="test_case" time="1.115" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.182" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.074" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.110" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.261" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.285" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.083" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.106" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.372" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.108" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.100" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.262" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.023" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.017" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.287" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.022" classname="com.portal.base.testCase.All_Test_Case">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="Expected to find an object with property [&#039;code&#039;] in path $ but found &#039;java.lang.String&#039;. This is not a json object according to the JsonProvider: &#039;com.jayway.jsonpath.spi.json.JsonSmartJsonProvider&#039;.">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['code'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
	at com.jayway.jsonpath.internal.path.PropertyPathToken.evaluate(PropertyPathToken.java:71)
	at com.jayway.jsonpath.internal.path.RootPathToken.evaluate(RootPathToken.java:62)
	at com.jayway.jsonpath.internal.path.CompiledPath.evaluate(CompiledPath.java:53)
	at com.jayway.jsonpath.internal.path.CompiledPath.evaluate(CompiledPath.java:61)
	at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
	at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:102)
	at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:85)
	at com.jayway.jsonpath.JsonPath.read(JsonPath.java:488)
	at com.portal.base.utils.AssertUtils.assertRespKeyInfo(AssertUtils.java:45)
	at com.portal.base.testCase.All_Test_Case.test_case(All_Test_Case.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- test_case -->
  <testcase name="test_case" time="0.225" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.163" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.069" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.202" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.083" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.032" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.212" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.067" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.099" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.248" classname="com.portal.base.testCase.All_Test_Case">
    <error type="java.lang.AssertionError" message="expected [20.76] but found [20.76]">
      <![CDATA[java.lang.AssertionError: expected [20.76] but found [20.76]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at com.portal.base.utils.AssertUtils.assertRespKeyInfo(AssertUtils.java:47)
	at com.portal.base.testCase.All_Test_Case.test_case(All_Test_Case.java:56)
	at sun.reflect.GeneratedMethodAccessor22.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- test_case -->
  <testcase name="test_case" time="2.564" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.254" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="1.118" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.116" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="2.109" classname="com.portal.base.testCase.All_Test_Case"/>
  <testcase name="test_case" time="0.197" classname="com.portal.base.testCase.All_Test_Case"/>
</testsuite> <!-- com.portal.base.testCase.All_Test_Case -->
